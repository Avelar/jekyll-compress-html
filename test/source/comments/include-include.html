---
layout: compress
---

<p>One

{% include comments-three.html %}
