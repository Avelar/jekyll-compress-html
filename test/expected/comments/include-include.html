<p>One <div>Two</div>
